mmdet/datasets/pipelines/loading.py

```diff
@@ -76,6 +76,116 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class LoadMultiImagesFromMultiFiles(object):
+    """Load multi images from multi files.
+    Required keys are "img_prefix" and img_info_keys (dicts that must contain
+    the key "filename"). Added or updated keys are "*filename*",
+    "*ori_filename*", "*img*", "img_shape", "ori_shape" (same as `img_shape`),
+    "pad_shape" (same as `img_shape`), "scale_factor" (1.0) and "img_norm_cfg"
+    (means=0 and stds=1).
+
+    Args:
+        img_info_keys (list): the keys about img info.
+            Defaults: ['img_info', 'ref_img_info']
+        to_float32 (bool): Whether to convert the loaded image to a float32
+            numpy array. If set to False, the loaded image is an uint8 array.
+            Defaults to False.
+        color_type (str): The flag argument for :func:`mmcv.imfrombytes()`.
+            Defaults to 'color'.
+        file_client_args (dict): Arguments to instantiate a FileClient.
+            See :class:`mmcv.fileio.FileClient` for details.
+            Defaults to ``dict(backend='disk')``.
+    """
+
+    def __init__(self,
+                 img_info_keys=['img_info', 'ref_img_info'],
+                 to_float32=False,
+                 color_type='color',
+                 file_client_args=dict(backend='disk')):
+        self.img_info_keys = img_info_keys
+        self.to_float32 = to_float32
+        self.color_type = color_type
+        self.file_client_args = file_client_args.copy()
+        self.file_client = None
+
+    def load_img_core(self, results, fname_id, fname, ori_fname, img_info_key):
+        img_bytes = self.file_client.get(fname)
+        img = mmcv.imfrombytes(img_bytes, flag=self.color_type)
+        if self.to_float32:
+            img = img.astype(np.float32)
+
+        # e.g. img_postfix = '' or 'ref_'
+        img_prefix = img_info_key[-12:-8]
```

Comment:
What if there are unexpected `img_info_key` passed to the module?

Reply:
line 151-152 will continue.

Reply:
I mean that user could specify the `self.img_info_keys` through config, and thus it might not be 'img_info', 'ref_img_info', but `img_prefix = img_info_key[-12:-8]` use hardcode 12 & 8.